index.html

```diff
@@ -428,6 +433,34 @@ <h3>Constructor</h3>
           </li>
         </ol>
       </section>
+      <section data-dfn-for="WakeLock">
+        <h3><dfn>requestPermission()</dfn> static method</h3>
+        <p>
+          The <a>requestPermission()</a> method, when invoked, MUST run the
+          following steps. This algorithm resolves with either
+          "<a>granted</a>" or "<a>denied</a>".
+        </p>
+        <ol class="algorithm">
+          <li>
+            Let |promise| be a new <a>Promise</a> object.
```

Comment:
Let |promise| be a newly created <a>Promise</a>.

Reply:
Is that the most common way? I have seen both in specs

Reply:
You are right. https://www.w3.org/2001/tag/doc/promises-guide#a-new-promise 

So actually the correct one would be:

> Let |promise| be `<a data-cite="promises-guide#a-new-promise">a new promise</a>`.

Reply:
done